src/layouts/main.vue

```diff
@@ -1,6 +1,11 @@
 <template>
+  <div class="skipNav">
+    <a href="#mainContent" class="skip-main">Skip Navigation</a>
```

Comment:
Would Bootstrap's `sr-only` class work here?

Reply:
I should probably have been more specific in the description as these fixes support more than just visual impairments.  Typically, `skip navigation` links are there to support both users who use screenreaders as well as those using only a keyboard and have a physical condition that prevents them from using a mouse.  `sr-only` only addresses half of that population.

Reply:
@kentyunge I'm not very knowledgeable of these sorts of accessibility issues... I am trying to be more mindful of them though. Does [`sr-only-focusable`](https://bootstrapdocs.com/v3.3.5/docs/css/#helper-classes-screen-readers) address the other half?

Reply:
Probably - I didn't know bootstrap had that class built-in.  I'll test it out and push an update.